ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -111,6 +111,7 @@
   private int mFontStyle = ReactTypefaceUtils.UNSET;
   private boolean mAutoFocus = false;
   private boolean mDidAttachToWindow = false;
+  private Integer mMaximumTextLength = 0;
```

Comment:
Does this need to be Integer? The wrapped object Integer type has more overhead because the actual operations (less-than, etc) will first convert this to an `int`. Let's just use `int` here instead and below as well if possible?

Reply:
Thanks a lot. I'm reading changes from https://github.com/facebook/react-native/commit/de44184e01d74e4da18f1cba9b01d4c259ef5d9c and re-evaluating this solution.

Reply:
Sorry @JoshuaGross I added all the changes requested with https://github.com/facebook/react-native/pull/29070/commits/b86f4985ca25320301fd8196505ec5ca52e017d0 and I updated the PR summary with all the relevant information :pray: Thanks a lot